en/flight_controller/betafpv_beta75x.md

```diff
@@ -0,0 +1,33 @@
+# BetaFPV Beta75X 2S Brushless Whoop
+
+The [BetaFPV Beta75X](https://betafpv.com/products/beta75x-2s-whoop-quadcopter) is a very small quadrotor that can be flown indoors or outdoors, FPV or line-of-sight.
+
+![BetaFPV Beta75X](../../assets/hardware/betafpv_beta75x.jpg)
+
+## Where to Buy
+
+The *Beta75X* can be bought from a number of vendors, including:
+- [GetFPV](https://www.getfpv.com/beta75x-2s-brushless-whoop-micro-quadcopter-xt30-frsky.html)
```

Comment:
FYI, I just did this because I prefer to add multiple sources if there are any.

Reply:
Agreed. Here in Switzerland we even get them from local stores, but I don't think it makes sense to add those.